django/forms/widgets.py

```diff
@@ -27,21 +27,19 @@
     'SplitDateTimeWidget', 'SplitHiddenDateTimeWidget',
 )
 
-MEDIA_TYPES = ('css', 'js')
-
 
 @python_2_unicode_compatible
 class Media(object):
+    MEDIA_TYPES = (('css', dict), ('js', list))
```

Comment:
MEDIA_TYPES is not a module level constant anymore, I'd rename it to `media_types`.